test/setup_test.dart

```diff
@@ -89,8 +89,8 @@ void main() {
     for (final fen in [
       '8/8/8/8/8/8/8/8 w - - 1+2 12 42',
       '8/8/8/8/8/8/8/8[Q] b - - 0 1',
-      'r3k2r/8/8/8/8/8/8/R3K2R[] w Qkq - 0 1',
```

Comment:
The question is how to best represent empty pockets in a fen. The code already accurately loads both "[]" and "" into a Crazyhouse or Chess♯ game. I changed the fen output for empty pockets from "[]" to "" because if I take a Chess♯ endgame fen (after all the pieces have been placed) and copy it to the [Board Editor](https://lichess.org/editor) it will fail if it has the empty square brackets. So why not output the version with no brackets since it loads the same way into Chess♯/Crazyhouse, but also lets you get a Stockfish analysis.

After making that change, these empty square brackets were causing the tests to fail because the outputed fen no longer matched the inputted one.

Reply:
if the `"[]"` in the FEN fails in the lichess board editor, then I suppose it should be removed :)
looks more like an incompatibility between chessops and scalachess